task/buildah-dance/0.1/buildah-dance.yaml

```diff
@@ -0,0 +1,253 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: "docker"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "image-build, appstudio, hacbs"
```

Comment:
Do we need to continue pulling the `appstudio` and `hacbs` tags?

Reply:
I'm really not sure what these tags do nowadays. Maybe @mmorhun can help.

Reply:
Probably nothing? It might be nice to have a common tag for all RHTAP (former Dance) tasks though.

Reply:
I'm still not sure what's the best approach here. Should we remove these and add a new `rhtap` tag? The tekton catalog only has `containers` as [tag](https://github.com/openshift-pipelines/tektoncd-catalog/blob/p/tasks/task-buildah/0.2.0/task-buildah.yaml#L12).

Also, the min version seems off too.

Reply:
I'd remove outdated stuff. To me only `rhtap` makes sense. We might also need that `containers` for consistency.

Reply:
Ok, I changed the tags to `containers, rhtap`.